corehq/apps/translations/app_translations/utils.py

```diff
@@ -161,11 +161,20 @@ def is_single_sheet(identifier):
     return identifier == SINGLE_SHEET_NAME
 
 
-def get_menu_or_form(app, identifier):
-    if '_' in identifier:
-        return get_form_from_sheet_name(app, identifier)
+def get_menu_or_form_by_sheet_name(app, sheet_name):
+    if '_' in sheet_name:
+        return get_form_from_sheet_name(app, sheet_name)
     else:
-        return get_module_from_sheet_name(app, identifier)
+        return get_module_from_sheet_name(app, sheet_name)
+
+
+def get_menu_or_form_by_unique_id(app, unique_id, row_type):
+    if row_type == 'Form' or is_form_sheet(row_type):
+        return app.get_form(unique_id)
+    elif row_type == 'Menu' or is_module_sheet(row_type):
+        return app.get_module_by_unique_id(unique_id)
+    else:
+        raise ValueError('Unknown row type "{}"'.format(row_type))
```

Comment:
translate

Reply:
Dropping `row_type` to use `sheet_name` instead. Also using consistent error messages regardless of whether we use `unique_id` or `sheet_name` to fetch the module/form.